inbm/dispatcher-agent/dispatcher/source/constants.py

```diff
@@ -26,22 +26,22 @@ class SourceParameters:
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationAddSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
+    sources: List[str] = field(default_factory=list)
     gpg_key_uri: Optional[str] = field(default=None)
     gpg_key_name: Optional[str] = field(default=None)
 
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationRemoveSourceParameters:
-    file_name: str
+    source_list_file_name: str
     gpg_key_name: Optional[str] = field(default=None)
 
 
 @dataclass(kw_only=True, frozen=True)
 class ApplicationUpdateSourceParameters:
-    file_name: str
-    sources: list[str] = field(default_factory=lambda: [])
+    source_list_file_name: str
```

Comment:
**Suggestion:** The 'source_list_file_name' attribute has been added to multiple classes. If this attribute is common across classes, consider using a base class or a mixin to avoid repetition. [maintainability]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    @dataclass(frozen=True)
    class SourceListFileMixin:
        source_list_file_name: str

    @dataclass(frozen=True)
    class ApplicationAddSourceParameters(SourceListFileMixin):
        ...

    @dataclass(frozen=True)
    class ApplicationRemoveSourceParameters(SourceListFileMixin):
        ...

    @dataclass(frozen=True)
    class ApplicationUpdateSourceParameters(SourceListFileMixin):
        ...
```